editor/code_editor.h

```diff
@@ -190,6 +191,8 @@ class CodeTextEditor : public VBoxContainer {
 	void _notification(int);
 	static void _bind_methods();
 
+	bool _is_warnings_panel_opened;
```

Comment:
We don't prefix private/protected members with an underscore usually, only methods.

Reply:
Fixed!